src/libraries/Microsoft.PowerFx.Core/Public/Types/AggregateType.cs

```diff
@@ -12,41 +12,60 @@ namespace Microsoft.PowerFx.Types
 {
     public abstract class AggregateType : FormulaType
     {
+        public IEnumerable<string> FieldNames { get; }
+
+        public ITypeIdentity Identity { get; }
```

Comment:
>ITypeIdentity [](http://example.com/codeflow?start=15&length=13)

Can this be null? (for a known type) 
What does this mean for a table? (ie, is there some implied type algebra)